salvo/src/lib/cmd_exec.py

```diff
@@ -70,3 +70,28 @@ def run_command(cmd: str, parameters: CommandParameters) -> str:
 
   return output
 
+def run_check_command(cmd: str, parameters: CommandParameters) -> None:
+  """Run the specified command checking its exit status. If the command exits
```

Comment:
(nit) The second sentence here ("If the command exits ..."), doesn't really belong into the one line docstring since it describes a raise. The Raises section is more suited to do that.

Reply:
I will update this.